packages/block-editor/src/components/block-list/block.js

```diff
@@ -394,10 +394,6 @@ function BlockListBlock( {
 					animationStyle
 			}
 		>
-			{ shouldRenderDropzone && <BlockDropZone
-				clientId={ clientId }
-				rootClientId={ rootClientId }
-			/> }
```

Comment:
The main  problem with this PR I think is that the dropzone is smaller than before. In master for paragraphs, its size is "37px" with this pr it is "30px". They both suffer from the same issue, there's a space (margin) between blocks where you can't drop blocks. It's is more visible in this PR because of the smaller size though.

Reply:
@youknowriad I fixed the issue entirely by rewriting `useBlockDropZone` to work on `BlockList` instead of `BlockListBlock`. Now there are no longer "dead zones" between the blocks where you can't drop.

Reply:
This is cool, I'll test a bit later.

Reply:
Thanks! I think the error might be gone too, but let me know if it isn't. :)